core/src/main/java/com/sequenceiq/cloudbreak/reactor/handler/cluster/RotateSaltPasswordHandler.java

```diff
@@ -46,7 +47,17 @@ protected Selectable doAccept(HandlerEvent<RotateSaltPasswordRequest> event) {
         Long stackId = event.getData().getResourceId();
         try {
             StackDto stack = stackDtoService.getById(stackId);
-            rotateSaltPasswordService.rotateSaltPassword(stack);
+            RotateSaltPasswordType rotateSaltPasswordType = event.getData().getType();
+            switch (rotateSaltPasswordType) {
+                case SALT_BOOTSTRAP_ENDPOINT:
+                    rotateSaltPasswordService.rotateSaltPassword(stack);
+                    break;
+                case FALLBACK:
+                    rotateSaltPasswordService.rotateSaltPasswordFallback(stack);
+                    break;
```

Comment:
could you add logs before calling `rotateSaltPasswordService`? just to make it easy to find out whats happening